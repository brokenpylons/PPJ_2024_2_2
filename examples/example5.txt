var a = 1;
print(a);
var a = 2;
print(a);
