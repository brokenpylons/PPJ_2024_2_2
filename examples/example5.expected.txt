real("2") minus("-") real("1") minus("-") real("1") 
